source/whats-new.txt

```diff
@@ -21,6 +22,28 @@ Learn what's new in:
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _version-4.7.1:
+
+What's New in 4.7.1
+-------------------
+
+The 4.7.1 driver patches a potential data corruption bug in
+Client-Side Field Level Encryption (CSFLE) and Queryable Encryption (QE)
+that occurs when rotating :ref:`Data Encryption Keys <csfle-key-architecture>`
+(DEKs) encrypted with a :ref:`Customer Master Key <csfle-key-architecture>` (CMK)
+hosted on Google Cloud Key Management Service (GCP KMS) or Azure
+Key Vault.
```

Comment:
Suggestion: keep the info about the "data corruption bug" next to the information of what it is.

E.g. (paraphrased suggestion for brevity)
"The v4.7.1 driver fixes a bug that impacts CSFLE and QE... The bug can cause data corruption when rotating DEKs encrypted with CMKs hosted on..."

Reply:
Agree this improves the note, nice edit.